add log for variant level proposal check config (#278)
apollographql/apollo-studio-community: CHANGELOG.md

[comment]: <> "NOTE! Ensure all images are added via the \[label\]\(link\) syntax!"

## 2024-06-28
| What's new | |
| :--------- | :-: |

> | Configuring proposal check severity at the variant level is now enabled. Find this in Checks Config > This Variant > Proposals. With this, you can configure proposal checks to `Error` on a dev variant, but `Warn` on a Prod variant to allow hot fixes to go through, for example. | ![image](https://github.com/apollographql/apollo-studio-community/assets/1314446/c05c18cc-016a-47e4-b668-999cdd22b3ce)

Reviewer: ```suggestion
| Proposal check severity can now be configured at the variant level. Find this in Checks Config > This Variant > Proposals. With this, you can configure proposal checks to `Error` on a dev variant, but `Warn` on a Prod variant to allow hot fixes to go through, for example. | ![image](https://github.com/apollographql/apollo-studio-community/assets/1314446/c05c18cc-016a-47e4-b668-999cdd22b3ce)
```
